aiohomekit/controller/coap/connection.py

```diff
@@ -0,0 +1,971 @@
+
+import asyncio
+import struct
+import logging
+import tlv8
```

Comment:
I'm guessing this is https://pypi.org/project/tlv8/ and not aiohomekit/aiohomekit/protocol/tlv.py?

`tlv8` was forked (and improved) from homekit_python. aiohomekit was also forked from homekit_python as my asyncio changes were not really a good fit upstream. But my fork predates the work on homekit_python to pull out tlv8. So here we are currently stuck with whats in aiohomekit/protocol/tlv.py from homekit_python.

We also have https://github.com/Jc2k/aiohomekit/blob/5902621ca0a1c8bd4399fcf8f2b7767d880cc252/aiohomekit/model/characteristics/structs.py for decoding complicated nested structs. You can do something like `SelectedRTPStreamConfiguration.decode(bytes)`. See `aiohomekit/tlv8.py`. 

Don't want to have 2 tlv8 libraries on the go at once, so for now better to stick with whats available in the code base. We can look at moving to tlv8 seperately if its worth it.

Reply:
Sounds good, I'll work to pull tlv8 out of this PR

Reply:
tlv8 has been removed